File: 1/AuP_La07/AuP_La0000.cpp
/******************************************************************************************************
basic prog
*___________*
_*_________*_

*******************************************************************************************************/

/// Autors: Vladislavs Agarkovs
/// Veidots: 13.09.2022

#include <iostream>

int main() {

  int ok;
  int n;
  do {
    std::cin >> n;


    for (int i = 0; i < n; i++) {
      int l = i % 14;

      for (int j = 0; j < n; j++) {

        int k = j % 14;

        if ((k - l) == 0) {
          std::cout << "*";
        } else if (12-(k + l) == 0) {
          std::cout << "*";
        } else {
          std::cout << "_";
        }
      }
      std::cout << "\n";
    }

    std::cout << "Vai turpināt (1) vai beigt (0)?" << std::endl;
    std::cin >> ok;
  } while (ok == 1);

  return 0;
}
/***********************  Testu plāns ***********************
       output asd
*************************************************************/

File: 1/AuP_La06/AuP_La0000.py
 
#AuP_La0201.Izveidot C++ programmu,kura pārvērš Fārenheita grādos uzdotu temperatūru par temperatūru Celsija grādos
#C = 5 / 9(F - 32)
#Jābūt iespējai programmu izpildīt atkārtoti, neizejot no programmas.

#Autors : Vladislavs Agarkovs
#Veidots : 13.09.2022

def lielakais(a,b):
    if a > b:
        return a
    else:
        return b

while True:
   
    print("Ievadiet elementu virkni ")
    try:
        max = int(input())
        while True:
            n = int(input())
            max = lielakais(max,n)
          
    # if the input is not-integer, just print the list
    except:
        print("maksimalais skaitlis ir",max)
    
    ok = int(input("Vai turpināt (1) vai beigt (0)?\n"))
    if ok == 0:
        break


######################## Testu plāns ########################
#
#
#
#
#
#############################################################
